packages/siop-oid4vp/lib/authorization-response/PresentationExchange.ts

```diff
@@ -104,7 +113,7 @@ export class PresentationExchange {
     } else if (!this.allVerifiableCredentials || this.allVerifiableCredentials.length == 0) {
       throw new Error(`${SIOPErrors.COULD_NOT_FIND_VCS_MATCHING_PD}, no VCs were provided`)
     }
-    const selectResults: SelectResults = this.pex.selectFrom(presentationDefinition, this.allVerifiableCredentials, {
+    const selectResults: SelectResults = this.pex.selectFrom(presentationDefinition, this.removeMDocCredentials(this.allVerifiableCredentials), {
```

Comment:
Please make comments if you make these types of exceptions as to why it is needed

Reply:
done